src/App.js

```diff
@@ -1,11 +1,104 @@
 
 import './App.css';
+import { VideoDetailPage, Explore, SearchResult, History, VideosInPlaylist, AllPlaylists, LikedVideos, WatchLater, HomePage, Nav, Footer, ForgotPasswordPage, Login, PrivateRoute, Profile, ProfilePage, Settings, SignUp, ErrorPage} from "./Components";
+
+import {
+  Routes,
+  Route
+} from "react-router-dom";
+import { useEffect } from 'react';
+import axios from 'axios';
+import { useAuthentication, useStateContext } from './Context';
+
 
 export function App() {
+  const {dispatch} = useStateContext();
+  const {isUserLoggedIn, userId} = useAuthentication();
+
+  useEffect(()=>{
+
+    (async()=>{
+      try {
+        const { data: {response} } = await axios.get("https://uandistoreapi.herokuapp.com/videos");
+        dispatch({type: "SET_VIDEOS", payload: response})
+      } catch(error) {
+        console.log(error);
```

Comment:
Remove console. You can use toast.error here

Reply:
Done